pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusCommunityDepAnalyzer.java

```diff
@@ -231,7 +233,7 @@ private Path generateQuarkusProject(Predicate<String> artifactIdSelector, String
                 .filter(artifactIdSelector)
                 .collect(Collectors.toList());
         String command = String.format(
-                "mvn -X io.quarkus:quarkus-maven-plugin:%s:create -DprojectGroupId=tmp -DprojectArtifactId=tmp "
+                "mvn -X -Dmaven.test.skip=true io.quarkus:quarkus-maven-plugin:%s:create -DprojectGroupId=tmp -DprojectArtifactId=tmp "
```

Comment:
`-Dmaven.test.skip=true` seems unnecessary here

Reply:
I think it's necessary otherwise project creation fails. 
`quarkus-kafka-streams-extension` require specific config in application.properties file which is not there at the time of project creation. 

And it was actually test which was failing. So skipping tests actually solves the issue for us.

Reply:
are you sure it's `:create` goal that fails? That would mean the goal is unusable

Reply:
yes thats the goal that fails

Reply:
The create goal works without skipping tests. We required to skip tests only building the project. I tested it locally and it works

Reply:
wierd it failed on my local :\

Reply:
Done !